jep/0000/README.adoc

```diff
@@ -209,52 +209,20 @@ There are no backwards compatibility concerns related to this proposal.
 No existing permission types are being removed at this point, which will allow for existing plugins that make use of them to continue to function.
 Removing code references to the "dangerous" permissions is outside the scope of this JEP.
 
-==== Which permissions should Jenkins.CONFIGURE imply
+=== Jenkins.CONFIGURE implies no other permission
 
-We have considered which other permissions should be implied by `Jenkins.CONFIGURE`
+For the sake of customization, we are designing this permission to not imply other permissions that can be granted separately, as long as they are not needed for `Jenkins.CONFIGURE` 's main purpose (see <<Motivation>>).
 
-We are hesitating about `Jenkins.READ` (a.k.a `Overall Read`). 
+Hence, it will not imply any `Job`, `View` or `Agent` level permission.
 
-This permission is required to access the web or use the CLI, and is only implied by `Jenkins.ADMINISTER`, which means that currently any other permission allows to access the web, and it has to be given explicitly.
+About `Jenkins.READ` (a.k.a `Overall Read`, required to access the web or use the CLI), we have decided to *not* imply it for the following reasons:
 
-However, it is reasonable for `Jenkins.CONFIGURE`, as a permission of Jenkins/Overall scope, to imply it.
-
-At the same time, the current implementation of `Permission` makes impossible to make `Jenkins.CONFIGURE` imply `Jenkins.READ`, because a permission can not be implied by more than one, and `Jenkins.READ` is already implied by `Permission.READ`
+* To follow the current standard: Every other permission (except `Jenkins.ADMINISTER`) is not implying it. 
+* Technical limitation: the current implementation of `Permission` makes impossible to make `Jenkins.CONFIGURE` imply `Jenkins.READ`, because a permission can not be implied by more than one, and `Jenkins.READ` is already implied by `Permission.READ`. 
 
 image::CONFIGURE-and-READ-permissions.svg[]
 
-These are the different solutions we are considering:
-
-. `Jenkins.CONFIGURE` is not enough to access the website. 
```

Comment:
I think we should warn that the Jenkins.CONFIGURE permission will need Jenkins.READ to be able to navigate on the UI.

Reply:
oops it's after :)

Reply:
forget my comment